projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/boost-speed-score.tsx

```diff
@@ -0,0 +1,139 @@
+import { getScoreLetter, requestSpeedScores } from '@automattic/jetpack-boost-score-api';
+// We'll need the IconTooltip in a follow-up PR. TODO: remove me
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { IconTooltip, Spinner, BoostScoreBar } from '@automattic/jetpack-components';
+import { __, sprintf } from '@wordpress/i18n';
+import React, { useCallback, useEffect, useState } from 'react';
+import useAnalytics from '../../../hooks/use-analytics';
+import './style.scss';
+
+const BoostSpeedScore = () => {
+	const { recordEvent } = useAnalytics();
+	const [ isLoading, setIsLoading ] = useState( false );
+	const [ speedLetterGrade, setSpeedLetterGrade ] = useState( 'C' );
```

Comment:
Should we have a default value here? I know we'll never see it because the scorebar isn't shown until all the data is loaded, but that makes me think we can probably just make this an empty string? I don't think we'd want to show an incorrect score if there is any possibility the user sees it before the data is loaded

Reply:
Yes, very good point!  Making it an empty string. 👍